storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/JdbcHistoricalRetriever.java

```diff
@@ -138,11 +155,31 @@ private String exportResultsToDisk(Connection conn, String resultTable, String s
     String exportTableSqlQuery = null;
     try {
       Statement statement = conn.createStatement();
-      exportTableSqlQuery =
-          String.format(
-              "COPY %s TO '%s' WITH (DELIMITER E'\t', FORMAT CSV, HEADER);",
-              resultTable, exportPath);
-      statement.executeUpdate(exportTableSqlQuery);
+      if (this.className == "net.snowflake.client.jdbc.SnowflakeDriver") {
```

Comment:
All the conditionals will be removed and code will be in specific querytemplater in the next PR right?

Reply:
that's correct